src/QuickAddModal.tsx

```diff
@@ -1,21 +1,29 @@
 import React from "react";
```

Comment:
This file is getting a bit complicated. We might want to clean this up (split into smaller helper components/files?) at some point.